packages/playground/_stories/fiori/DynamicSideContent/DynamicSideContent.stories.ts

```diff
@@ -23,190 +27,107 @@ export default {
-					</div>
-				</ui5-dynamic-side-content>
-			</ui5-page>
-		</div>
-	</div>
-`;
-
-
-export const Template7: StoryFn = () => html`
-<h3>Dynamic Side Content - toggle contents on mobile device (S size)</h3>
-	<div class="snippet">
-		<div class="wrapperM">
-			<ui5-page floating-footer="" show-footer="">
-				<ui5-dynamic-side-content id="dynamicSideContent" class="content-padding">
-					<div>
-						<h1>Main Content</h1>
-						<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc ex mi, elementum et ante commodo, semper sollicitudin magna. Sed dapibus ut tortor quis varius. Sed luctus sem at nunc porta vulputate. Suspendisse lobortis arcu est, quis ultrices ipsum fermentum a. Vestibulum a ipsum placerat ligula gravida fringilla at id ex. Etiam pellentesque lorem sed sagittis aliquam. Quisque semper orci risus, vel efficitur dui euismod aliquet. Morbi sapien sapien, rhoncus et rutrum nec, rhoncus id nisl. Cras non tincidunt enim, id eleifend neque.</p>
-					</div>
-					<div slot="sideContent">
-						<h1>Side Content</h1>
-						<p>Morbi lorem libero, imperdiet id condimentum ac, tempor ut velit. Integer a laoreet sem. Nunc at sagittis nisi. Sed placerat diam eu porttitor dignissim. Maecenas nec fringilla tortor. Pellentesque ut elit est. Curabitur quis elit at mauris ullamcorper fringilla. Nullam diam mi, porttitor dictum orci nec, molestie luctus metus. Nunc ut ex blandit, elementum erat eget, pulvinar sapien. Donec nec lorem eu nunc eleifend tempor at non tortor. In quam velit, ornare at rutrum ac, porta ac augue. Suspendisse venenatis semper lacus at venenatis. Praesent vestibulum ligula nulla, at tempus lorem consequat suscipit. Aenean consequat dapibus dui, at bibendum mauris porta a.</p>
-					</div>
-				</ui5-dynamic-side-content>
-				<div slot="footer">
-					<ui5-bar design="FloatingFooter">
-						<ui5-button id="toggleButton" design="Positive" slot="endContent">Toggle</ui5-button>
-					</ui5-bar>
-				</div>
-			</ui5-page>
-		</div>
-	</div>
-	<script>
-		toggleButton.addEventListener("click", function() {
-			dynamicSideContent.toggleContents();
-		});
-	</script>
-`;
-Template7.parameters = {
-	docs: {
-		story: {
-			// Opt-out of inline rendering
-			inline: false,
-		},
-	}
+const Template: UI5StoryArgs<DynamicSideContent, StoryArgsSlots> = (args) => html`<ui5-dynamic-side-content
```

Comment:
This can be reduced to only 1 samle. the basic one, everything else can be achieved with setting the booleans